tests/test_wildcardmanager.py

```diff
@@ -90,15 +93,28 @@ def test_directory_traversal(wildcard_manager: WildcardManager):
     assert not wildcard_manager.get_all_values("..\\cant_touch_this")
 
 
-def test_clean_wildcard():
+def test_clean_wildcard(wildcard_manager: WildcardManager):
+    clean = partial(_clean_wildcard, wildcard_wrap=wildcard_manager.wildcard_wrap)
     with pytest.raises(ValueError):
-        _clean_wildcard("/foo")
+        clean("/foo", wildcard_wrap=wildcard_manager.wildcard_wrap)
 
     with pytest.raises(ValueError):
-        _clean_wildcard("\\foo")
+        clean("\\foo", wildcard_wrap=wildcard_manager.wildcard_wrap)
 
     with pytest.raises(ValueError):
-        _clean_wildcard("foo/../bar")
+        clean("foo/../bar", wildcard_wrap=wildcard_manager.wildcard_wrap)
```

Comment:
Isn't the `wildcard_wrap` arg here unnecessary since we already have the partial?